hyperparameter_tuning/experiment.py

```diff
@@ -54,9 +54,9 @@ def get_experiment_result(experiment_tunables):
         output (str): Result received from the experiment manager.
     """
     for tunable in experiment_tunables:
-        if tunable["tunable_name"] == "cpuRequest":
+        if tunable["tunable_name"] == "cpu_request":
```

Comment:
This needs to be inline with the current [codebase](https://github.com/kruize/autotune/blob/master/manifests/autotune-configs/container-config.yaml), so this needs to be `cpuRequest` and `memoryRequest`. We can have a separate PR for changing this across the entire codebase.